.CI/travis/install_osx.sh

```diff
@@ -5,7 +5,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-brew update
+#brew update
```

Comment:
Why is it commented and not deleted?

Reply:
I was just testing, if it works without that step, It will be deleted once all is working.